tracer/src/Datadog.Trace/DuckTyping/DuckType.Methods.cs

```diff
@@ -383,7 +383,7 @@ private static void CreateReverseProxyMethods(TypeBuilder? proxyTypeBuilder, Typ
                 }
 
                 Type[] parameterTypes = proxyMethodDuckAttributeParameterTypeNames
-                                                                .Select(pName => Type.GetType(pName, throwOnError: false))
```

Comment:
Actually, do we have a good reason we were using `false` here? 😟 It feels like we might actually be hiding strange issues which we won't know about in that case...

Reply:
Yeah because that change can be really impactful, I'll try to do that in another PR, is a change of behavior of the SelectMethod